charts/cluster-addons/templates/monitoring/loki.yaml

```
@@ -0,0 +1,123 @@
+{{- if and .Values.monitoring.enabled .Values.monitoring.loki_v2.enabled }}
+# TODO: Update this to include Loki v3 once the new addon is available
+{{- if and .Values.monitoring.lokiStack.enabled .Values.monitoring.loki_v2.enabled }}
+{{- fail (printf "Cannot enable both 'lokiStack' and 'loki' addons simultaneously; please choose one.") }}
+{{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "cluster-addons.componentName" (list . "loki") }}-config
+  labels:
+    {{- include "cluster-addons.componentLabels" (list . "loki") | nindent 4 }}
+    addons.stackhpc.com/watch: ""
+stringData:
+  defaults: |
+    serviceMonitor:
+      enabled: true
+    monitoring:
+      dashboards:
+        enabled: false
```

According to https://artifacthub.io/packages/helm/grafana/loki/5.48.0?modal=values&path=monitoring.serviceMonitor the `serviceMonitor` section should be under the `monitoring` key:
```suggestion
  defaults: |
    monitoring:
      serviceMonitor:
        enabled: true
      dashboards:
        enabled: false
```